src/main/navigation/navigation.h

```diff
@@ -272,7 +272,7 @@ typedef enum {
     MW_NAV_STATE_WP_ENROUTE,              // WP Enroute
     MW_NAV_STATE_PROCESS_NEXT,            // Process next
     MW_NAV_STATE_DO_JUMP,                 // Jump
-    MW_NAV_STATE_LAND_START,              // Start Land
+    MW_NAV_STATE_EMERGENCY_LANDING,       // Emergency landing
```

Comment:
Do we need to keep compatibility with Multiwii in iNav 2.0 ? Should I add a new state then and mark `MW_NAV_STATE_LAND_START` as unused ? Because `MW_NAV_STATE_LAND_START` doesn't describe the state accurately.

Reply:
Yes please mark START_LAND as unused and add a new enum. This more helpful to thrid party consumers that support both iNav and MW.